### PR TITLE
{tools}[foss/2018a,foss/2020a,intel/2018a] ToFu v1.4.6 w/ Python 3.6.4 + 3.8.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-foss-2018a-Python-3.6.4.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }),
+    ('tofu', version, {
+        'buildcmd': 'clean',
+        'use_pip': False,
+        'checksums': [
+            'ad31ba1198808a99d4c849280b18e382567317d19c1ca957d08da17a3441ed39',  # tofu-1.4.6.tar.gz
+        ],
+    }),
+]
+
```
Comment (lexming) [inline easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-foss-2018a-Python-3.6.4.eb:44]:
```suggestion

sanity_pip_check = True
```

File: easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-foss-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }),
+    ('tofu', version, {
+        'buildcmd': 'clean',
+        'use_pip': False,
+        'checksums': [
+            'ad31ba1198808a99d4c849280b18e382567317d19c1ca957d08da17a3441ed39',  # tofu-1.4.6.tar.gz
+        ],
+    }),
+]
+
```
Comment (lexming) [inline easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-foss-2020a-Python-3.8.2.eb:45]:
```suggestion

sanity_pip_check = True
```

File: easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-intel-2018a-Python-3.6.4.eb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    }),
+    ('tofu', version, {
+        'buildcmd': 'clean',
+        'use_pip': False,
+        'checksums': [
+            'ad31ba1198808a99d4c849280b18e382567317d19c1ca957d08da17a3441ed39',  # tofu-1.4.6.tar.gz
+        ],
+    }),
+]
+
```
Comment (lexming) [inline easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-intel-2018a-Python-3.6.4.eb:44]:
```suggestion

sanity_pip_check = True
```

Comment (SimonPinches) [reply easybuild/easyconfigs/t/ToFu/ToFu-1.4.6-intel-2018a-Python-3.6.4.eb:44]: @lexming Easyconfigs updated as suggested.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/32447f8106aa6a73accd1fb10a1ed4f2 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node111.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/f1ab7754ada1b808657a941eacad61f2 for a full test report.

**Comment by boegel** [comment]
Closing this, since the `foss/2018a` toolchain is now [deprecated](https://docs.easybuild.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains), and we have easyconfigs for more recent version of ToFu already (see #15328 and #15143)

**Pull request closed without merging**